{"hierarchy":{"paths":[["doc:\/\/com.github.pulse\/documentation\/Pulse","doc:\/\/com.github.pulse\/documentation\/Pulse\/RemoteLogger","doc:\/\/com.github.pulse\/documentation\/Pulse\/RemoteLogger\/ConnectionError"]]},"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/com.github.pulse\/documentation\/Pulse\/RemoteLogger\/ConnectionError\/unknown(isProtected:)"},"schemaVersion":{"patch":0,"minor":3,"major":0},"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"unknown"},{"kind":"text","text":"("},{"text":"isProtected","kind":"externalParam"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","text":"Bool","preciseIdentifier":"s:Sb"},{"text":")","kind":"text"}],"platforms":["iOS"],"languages":["swift"]}]}],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/pulse\/remotelogger\/connectionerror\/unknown(isprotected:)"]}],"sections":[],"metadata":{"role":"symbol","fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"unknown"},{"text":"(","kind":"text"},{"kind":"externalParam","text":"isProtected"},{"text":": ","kind":"text"},{"text":"Bool","kind":"typeIdentifier","preciseIdentifier":"s:Sb"},{"text":")","kind":"text"}],"modules":[{"name":"Pulse"}],"symbolKind":"case","roleHeading":"Case","externalID":"s:5Pulse12RemoteLoggerC15ConnectionErrorO7unknownyAESb_tcAEmF","title":"RemoteLogger.ConnectionError.unknown(isProtected:)"},"kind":"symbol","references":{"doc://com.github.pulse/documentation/Pulse":{"role":"collection","url":"\/documentation\/pulse","kind":"symbol","title":"Pulse","abstract":[{"type":"text","text":"Logger and network inspector for Apple platforms."}],"identifier":"doc:\/\/com.github.pulse\/documentation\/Pulse","type":"topic"},"doc://com.github.pulse/documentation/Pulse/RemoteLogger/ConnectionError/unknown(isProtected:)":{"role":"symbol","type":"topic","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"unknown","kind":"identifier"},{"kind":"text","text":"("},{"text":"isProtected","kind":"externalParam"},{"kind":"text","text":": "},{"kind":"typeIdentifier","preciseIdentifier":"s:Sb","text":"Bool"},{"text":")","kind":"text"}],"abstract":[],"title":"RemoteLogger.ConnectionError.unknown(isProtected:)","identifier":"doc:\/\/com.github.pulse\/documentation\/Pulse\/RemoteLogger\/ConnectionError\/unknown(isProtected:)","kind":"symbol","url":"\/documentation\/pulse\/remotelogger\/connectionerror\/unknown(isprotected:)"},"doc://com.github.pulse/documentation/Pulse/RemoteLogger/ConnectionError":{"identifier":"doc:\/\/com.github.pulse\/documentation\/Pulse\/RemoteLogger\/ConnectionError","title":"RemoteLogger.ConnectionError","abstract":[],"role":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"ConnectionError"}],"fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"ConnectionError"}],"kind":"symbol","url":"\/documentation\/pulse\/remotelogger\/connectionerror"},"doc://com.github.pulse/documentation/Pulse/RemoteLogger":{"kind":"symbol","type":"topic","fragments":[{"text":"class","kind":"keyword"},{"text":" ","kind":"text"},{"text":"RemoteLogger","kind":"identifier"}],"identifier":"doc:\/\/com.github.pulse\/documentation\/Pulse\/RemoteLogger","title":"RemoteLogger","role":"symbol","url":"\/documentation\/pulse\/remotelogger","abstract":[{"type":"text","text":"Connects to the remote server and sends logs remotely. In the current version,"},{"type":"text","text":" "},{"text":"a server is a Pulse Pro app for macOS).","type":"text"}],"navigatorTitle":[{"text":"RemoteLogger","kind":"identifier"}]}}}